data/src/main/java/com/example/data/utils/CityConverter.kt

```diff
@@ -0,0 +1,37 @@
+
+package com.example.data.utils
+
+import android.content.Context
+import com.example.data.R
+import com.example.domain.models.CityModel
+import com.google.gson.Gson
+import java.io.BufferedReader
+import java.io.InputStreamReader
+
+fun getCityModelByCityId(context: Context, cityId: Int): CityModel {
```

Comment:
All these functions should be moved into separate class as usual (non static) functions. Otherwise it will be hard to write tests.

Reply:
Fixed